sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/TablesLiveTestBase.cs

```diff
@@ -8,20 +8,23 @@
 using System.Threading.Tasks;
 using Azure;
 using Azure.Core.TestFramework;
+using Azure.Core.TestFramework.Models;
 using Azure.Data.Tables;
 using Azure.Data.Tables.Tests;
 using Microsoft.Azure.WebJobs.Host;
 using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.Extensions.Azure;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
 using NUnit.Framework;
 
 namespace Microsoft.Azure.WebJobs.Extensions.Tables.Tests
 {
-    [TestFixture(true)]
-    [TestFixture(false)]
-    public class TablesLiveTestBase : LiveTestBase<TablesTestEnvironment>
+    [AsyncOnly]
```

Comment:
why only async?

Reply:
Because the extension is the one calling client methods and switching them to be sync doesn't make sense as it would never run like that in production.